### ISSUE TITLE
Add Keys to dotenv file

### ISSUE THREAD

**Issue by wouter-toppershops** [mollie/magento2#84]
An option to store the test and live keys to the `.env` file.

see releated pull request

**Comment by wouter-toppershops** [comment]
Add this to your `.env` 
```
CONFIG__DEFAULT__PAYMENT__MOLLIE_GENERAL__APIKEY_TEST=
CONFIG__DEFAULT__PAYMENT__MOLLIE_GENERAL__APIKEY_LIVE=
```

(add it somewhere to the readme or other)

**Comment by wouter-toppershops** [comment]
Note for if you dont know. if the parameters isnt set in the env, the original config is still editable ;-)

**Comment by Marvin-Magmodules** [comment]
Was just reading up on this. Will do some testing next week and wil get this merged into 1.4.0. Great, thnx!

**Comment by wouter-toppershops** [comment]
thanks, look forward to it ;-)

 i think 1.4 will also include Klarna?

**Comment by Marvin-Magmodules** [comment]
Hi @wouter-toppershops yes, it will! 
Pre-release is available here: https://github.com/mollie/magento2/tree/1.4.0

**Comment by wouter-toppershops** [comment]
hi @Marvin-Magmodules 

any updates?

**Comment by Marvin-Magmodules** [comment]
Hi @wouter-toppershops this will be implemented in the stable v1.4 which is still in beta, once this is done we will look into this and merge it.

**Comment by Frank-Magmodules** [comment]
Hi @wouter-toppershops, PR is merged and [released](https://github.com/mollie/magento2/releases/tag/v1.4.0). Thank again for the PR and sorry for the long wait, the release took a bit more time than anticipated.

**Comment by wouter-toppershops** [comment]
Thanks @Frank-Magmodules
